tests/ztest/include/ztest_log.h

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright (c) 2017 Intel Corporation
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/**
+ * @file
+ *
+ * @brief Zephyr testing suite
+ */
+
+#ifndef __ZTEST_LOG_H__
+#define __ZTEST_LOG_H__
+
+#if CONFIG_ZTEST_LOGGING
```

Comment:
we should just use SYS_LOG macros instead of introducing another set of logging infrastructure.